boofuzz/sessions.py

```diff
@@ -1162,6 +1171,10 @@ def transmit_fuzz(self, sock, node, edge, callback_data):
             else:
                 self._fuzz_data_logger.log_info(msg)
             pass
+        except exception.BoofuzzTargetConnectionFailedError as e:
+            self._fuzz_data_logger.log_fail(str(e))
+        except exception.BoofuzzSSLError as e:
+            self._fuzz_data_logger.log_fail(str(e))
```

Comment:
## Just minor stylish issue
When the block in exception handling is the same for more types of exception, you can catch multiple of them in one line.

In this example:
```python
except (exception.BoofuzzTargetConnectionFailedError, exception.BoofuzzSSLError ) as e:
    self._fuzz_data_logger.log_fail(str(e))
```

Src: https://docs.python.org/3/tutorial/errors.html#handling-exceptions

Reply:
Fixed in Commit fc997eb